tornado_stale_client/async_client.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of tornado-stale-client.
+# https://github.com/globocom/tornado-stale-client
+
+# Licensed under the MIT license:
+# http://www.opensource.org/licenses/MIT-license
+# Copyright (c) 2016, Globo.com <backstage@corp.globo.com>
+
+import logging
+
+from tornado import gen
+from tornado.httpclient import HTTPRequest
+
+from tornado_stale_client.base import BaseStaleHTTPClient
+
+
+class AsyncStaleHTTPClient(BaseStaleHTTPClient):
```

Comment:
No problem for me @ricobl.
I don't renamed the original for compatibility with projects that already use `tornado_stale_client`.

`AsyncRedisStaleHTTPClient` is better then `AsyncStaleHTTPClient` class name?
Do you suggest some another @ricobl?